sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -98,10 +98,15 @@ public TestRandom Random
                     switch (Mode)
                     {
                         case RecordedTestMode.Live:
+                            #if NET6_0_OR_GREATER
+                            var liveSeed = RandomNumberGenerator.GetInt32(int.MaxValue);
+                            #else
```

Comment:
Do these need to be merged into the Test Proxy as well?

Reply:
No, the SetVariable will be implemented to call into test-proxy but we'll keep the generation on our side.